sdk/appcenter/src/test/java/com/microsoft/appcenter/channel/DefaultChannelRaceConditionTest.java

```diff
@@ -41,16 +41,16 @@
 
 public class DefaultChannelRaceConditionTest extends AbstractDefaultChannelTest {
 
-    @Test
+    @Test(timeout = 5000)
```

Comment:
what is that for?

Reply:
In some stage it was failing - deadlock inside. The timeout helps test fail within a reasonable time